### PR TITLE
Add dev_compiler and frontend_server to package uploading rule

### PR DESCRIPTION
**Pull request by jonahwilliams**
To support locally built frontend servers for the flutter tool.

### CODE REVIEW

File: flutter_frontend_server/package_incremental.py [hunk 1]
```diff
@@ -84,13 +86,40 @@
...
+
 PUBSPECS = {
   'vm': VM_PUBSPEC,
   'build_integration': BUILD_INTEGRATION_PUBSPEC,
   'flutter_frontend_server': FLUTTER_FRONTEND_SERVER_PUBSPEC,
   'kernel': KERNEL_PUBSPEC,
   'front_end': FRONT_END_PUBSPEC,
   'dev_compiler': DEV_COMPILER_PUBSPEC,
+  'frontend_server': FRONTEND_SERVER_PUBSPEC,
+  'dev_compiler': DEV_COMPILER_PUBSPEC,
```
Comment (zanderso) [inline flutter_frontend_server/package_incremental.py]: This line looks the same as line 120

Comment (jonahwilliams) [reply flutter_frontend_server/package_incremental.py]: :man_facepalming:

File: flutter_frontend_server/package_incremental.py [hunk 2]
```diff
@@ -84,13 +86,27 @@
...
+environment:
+  sdk: '>=2.2.2 < 3.0.0'
+dependencies:
+  build_integration: any
+  vm: any
+  dev_compiler: any
+  front_end: any
+  kernel: any
+  args: any
+''';
```
Comment (zanderso) [inline flutter_frontend_server/package_incremental.py]: Is the semicolon needed?

**Comment by jonahwilliams** [comment]
Looks like I'm synced to a bad commit, will update

File: flutter_frontend_server/package_incremental.py (hunk 2 above)
Comment (jonahwilliams) [reply flutter_frontend_server/package_incremental.py]: Fixed

**Pull request merged**